README.md

```diff
@@ -1,45 +1,64 @@
 ![development-status](https://badgen.net/badge/development-status/alpha/yellow?icon=github)
 
-# Multiplayer Collaboration
+# Ably
 
-[Ably](https://ably.com/) is the platform that powers realtime experiences at scale, including live chat, data broadcast, notifications, audience engagement and collaboration.
+_[Ably](https://ably.com) is the platform that powers synchronized digital experiences in realtime. Whether attending an event in a virtual venue, receiving realtime financial information, or monitoring live car performance data – consumers simply expect realtime digital experiences as standard. Ably provides a suite of APIs to build, extend, and deliver powerful digital experiences in realtime for more than 250 million devices across 80 countries each month. Organizations like Bloomberg, HubSpot, Verizon, and Hopin depend on Ably’s platform to offload the growing complexity of business-critical realtime data synchronization at global scale. For more information, see the [Ably documentation](https://ably.com/docs)._
 
-Multiplayer Collaboration enables virtual teams to work together efficiently by having the contextual awareness of who else is currently with them on the app, where exactly they are and the updates they are making. With the Ably Spaces SDK, we are on a mission to make it easy for developers to add Multiplayer Collaboration features to any web application, in a few lines of code.
+## Collaborative Spaces SDK
+
+The Collaborative Spaces SDK enables you to implement realtime collaborative features in your applications. The requirement to have multiplayer collaboration in applications has only been increasing in recent years. Rather than having to coordinate resources on calls, or send documents and spreadsheets back and forth, features that make collaborating in realtime are becoming a necessity rather than a nice-to-have.
+
+The main three features that you need to make your application collaborative are:
+
+* Who is in the application?
+* Where is each user within the application?
+* What is everyone doing in the application?
 
 ![Multiplayer Collaboration Demo](https://user-images.githubusercontent.com/5900152/225328262-2b63bb49-57a0-4f40-b78e-b87565f4c98c.png)
 
-> **Note**
-> The **Ably Spaces SDK** is a work in progress. It is currently meant for early adopters to trial and test out while some features are still under development. You can try it out with the usage instructions below.
+### Who is in the application?
```

Comment:
I think it would be confusing to mix the who, where, what with the technical concepts of the space. This is because space could mean an entire collaborative application or one part of an entire application. For example, think Miro dashboard vs one miro board. Let's chat about clarifying this. I do think that we need to separate the marketing concepts above clearly with the technical concepts being introduced from here on.

Reply:
Let me know if this one is covered by my rewrite above or happy to chat through still!